src/explore-education-statistics-admin/src/pages/publication/components/PublicationContactForm.tsx

```diff
@@ -36,16 +36,17 @@ const PublicationContactForm = ({
       .required('Enter a team email')
       .email('Enter a valid team email'),
     contactName: Yup.string().required('Enter a contact name'),
-    contactTelNo: Yup.string().test({
-      name: 'telephone',
-      message: 'Enter a contact telephone',
-      test: value => {
-        if (!value) {
-          return false;
-        }
-        return true;
-      },
-    }),
+    contactTelNo: Yup.string()
+      .trim()
+      .matches(
+        /^[0-9 \t]*$/,
```

Comment:
It'd be more suitable to use `\s` as covers any type of whitespace (and is more succinct). ` \t` only covers space and tabs.